### PR TITLE
feat: handle cluster trial

### PR DESCRIPTION
**Pull request by ShahAnuj2610**

### CODE REVIEW

File: modules/reducers/utils.js [hunk 1]
```diff
@@ -30,6 +30,11 @@ export const computePlan = ({ payload }) => {
 		daysLeft: payload.tier_validity
 			? Math.ceil((payload.tier_validity - new Date().getTime() / 1000) / (24 * 60 * 60))
 			: 0,
+		clusterDaysLeft: payload.cluster_tier_validity
```
Comment (lakhansamani) [inline modules/reducers/utils.js]: @ShahAnuj2610 here we are already getting number of days so you don't need to calculate days here

Comment (ShahAnuj2610) [reply modules/reducers/utils.js]: days for the cluster are different than that for the app, it's `tier_validity` vs `cluster_tier_validity`

Comment (lakhansamani) [reply modules/reducers/utils.js]: Right but if you check response here, https://i.imgur.com/46jASGy.png

`cluster_tier_validity` is already a number of days remaining for trial, so you don't need to `(payload.cluster_tier_validity - new Date().getTime() / 1000) / (24 * 60 * 60)`

Comment (ShahAnuj2610) [reply modules/reducers/utils.js]: oh, I see but shouldn't the datatypes for both be same?

Comment (lakhansamani) [reply modules/reducers/utils.js]: Yes datatype would be same, but its just that you don't need to calculate

**Pull request merged**